meta-facebook/yv4-sd/src/platform/plat_pldm_monitor.c

```diff
@@ -22,15 +22,31 @@
 #include "pldm.h"
 
 #include "plat_gpio.h"
+#include "plat_i2c.h"
 #include "plat_sensor_table.h"
 #include "plat_pldm_monitor.h"
 
 LOG_MODULE_REGISTER(plat_pldm_monitor);
 
+#define POWER_ON_BUTTON_MSEC 1000
```

Comment:
Lets use PLAT_PLDM as prefix to keep it consistent through out code.

Reply:
Revised, thank you for the suggestion.